fe/src/main/java/org/apache/doris/rewrite/FEFunctions.java

```diff
@@ -421,4 +422,21 @@ public static LiteralExpr ifNullBigInt(LiteralExpr first, LiteralExpr second) th
     public static LiteralExpr ifNullDateTime(LiteralExpr first, LiteralExpr second) throws AnalysisException {
         return first instanceof NullLiteral ? second : first;
     }
+
+    /**
+     ------------------------------------------------------------------------------
+     */
+
+    /**
+     * Nest type function
+     */
+    @FEFunction(name = "array", argTypes = {"INT"}, returnType = "ARRAY")
+    public static LiteralExpr arrayInt(LiteralExpr... exprs) throws AnalysisException {
+        return new ArrayLiteral(exprs);
+    }
+
+    @FEFunction(name = "array", argTypes = {"VARCHAR"}, returnType = "ARRAY")
+    public static LiteralExpr array(StringLiteral... exprs) throws AnalysisException {
```

Comment:
better to call arrayString()?

Reply:
Em...the method is same with array(IntLiteral.....) in fact, maybe will same with array(BigintLiteral.....)、array(BoolLiteral.....) in future......I think better way is refactor the annotation for support a list of types